<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageTeam</class>
 <widget class="QWidget" name="pageTeam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" rowspan="2">
          <widget class="QLabel" name="labelTeamPix">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>队伍图片</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelTeamEn">
         <property name="text">
          <string>队伍名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>球队数据</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="3">
            <widget class="QPushButton" name="buttonVictory">
             <property name="text">
              <string>胜负</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="buttonPenalty">
             <property name="text">
              <string>罚球</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonScore">
             <property name="text">
              <string>分数</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="buttonFalut">
             <property name="text">
              <string>失误/犯规</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QPushButton" name="buttonBackboard">
             <property name="text">
              <string>篮板</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="buttonAssisting">
             <property name="text">
              <string>助攻/抢断/盖帽</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonShoot">
             <property name="text">
              <string>投篮</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="buttonThree">
             <property name="text">
              <string>三分</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>6</number>
           </property>
           <widget class="QWidget" name="pageShoot">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QGridLayout" name="layoutShoot"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageThree">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QGridLayout" name="layoutThree"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pagePenalty">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QGridLayout" name="layoutPenalty"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageBackboard">
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <layout class="QGridLayout" name="layoutBackboard"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageAssisting">
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QGridLayout" name="layoutAssisting"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageFalut">
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QGridLayout" name="layoutFalut"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageScore">
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QGridLayout" name="layoutScore"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageVictory">
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QGridLayout" name="layoutVictory"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>球队成员</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutPlayer"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
